nestkernel/simulation_manager.cpp

```diff
@@ -443,13 +499,17 @@ nest::SimulationManager::simulate( Time const& t )
       "is called repeatedly with simulation times that are not multiples of "
       "the minimal delay." );
 
-  resume_( num_active_nodes );
+  resume_();
+}
 
+void
+nest::SimulationManager::cleanup()
+{
   finalize_simulation_();
 }
 
 void
-nest::SimulationManager::resume_( size_t num_active_nodes )
+nest::SimulationManager::resume_()
 {
   assert( kernel().is_initialized() and not inconsistent_state_ );
 
```

Comment:
Same as before -- maybe. How much should we take advantage here to clean up code, and how much should we avoid making extra changes in this same set of commits?